tests/sentry/tasks/deletion/test_hybrid_cloud.py

```diff
@@ -274,3 +287,425 @@ def test_set_null_deletion_behavior(task_runner):
     # Deletion set field to null
     saved_query = DiscoverSavedQuery.objects.get(id=saved_query.id)
     assert saved_query.created_by_id is None
+
+
+def setup_cross_db_deletion_data(
+    desired_user_id: int | None = None,
+    desired_monitor_id: int | None = None,
+):
+    user = Factories.create_user(id=desired_user_id)
+    organization = Factories.create_organization(owner=user, name="Delete Me")
+    project = Factories.create_project(organization=organization)
+    group = Factories.create_group(project=project)
+    with assume_test_silo_mode_of(DiscoverSavedQuery, Monitor):
+        saved_query = DiscoverSavedQuery.objects.create(
+            name="disco-query",
+            organization=organization,
+            created_by_id=user.id,
+        )
+        monitor = Monitor.objects.create(
+            id=desired_monitor_id,
+            organization_id=organization.id,
+            project_id=project.id,
+            slug="test-monitor",
+            name="Test Monitor",
+            owner_user_id=user.id,
+        )
+
+        assert monitor.owner_user_id == user.id
+
+    return dict(
+        user=user,
+        organization=organization,
+        project=project,
+        monitor=monitor,
+        group=group,
+        saved_query=saved_query,
+    )
+
+
+# @region_silo_test
+# class TestCrossDatabaseTombstoneCascadeBehavior(TestCase):
+#     def assert_monitors_unchanged(self, unaffected_data: list[dict]):
+#         for u_data in unaffected_data:
+#             u_user, u_monitor = itemgetter("user", "monitor")(u_data)
+#             queried_monitor = Monitor.objects.get(id=u_monitor.id)
+#             # Validate that none of the existing user's monitors have been affected
+#             assert u_monitor.owner_user_id is not None
+#             assert u_monitor.owner_user_id == queried_monitor.owner_user_id
+#             assert u_monitor.owner_user_id == u_user.id
+#
+#     def assert_monitors_user_ids_null(self, monitors: list[Monitor]):
+#         for monitor in monitors:
+#             monitor.refresh_from_db()
+#             assert monitor.owner_user_id is None
+#
+#     def run_hybrid_cloud_fk_jobs(self):
+#         with override_options({"hybrid_cloud.allow_cross_db_tombstones": True}):
+#             with BurstTaskRunner() as burst:
+#                 schedule_hybrid_cloud_foreign_key_jobs()
+#
+#             burst()
+#
+#     def test_raises_when_option_disabled(self):
+#         data = setup_cross_db_deletion_data()
```

Comment:
These tests actually validate the cross-DB tombstone behavior, but rely entirely on PR #69697 in order to function correctly. Unfortunately, PR #69697 is also reliant on these fixes in order to pass testing 🐔/🥚 .

Reply:
Instead of checking in commented out code which can be misinterpreted as a mistake, you could use `pytest.mark.skip` to not run the tests but include them as well.